### DISCUSSION TITLE
Obsidian有中文帮助文档吗？

### DISCUSSION THREAD

**Discussion by lesswrongzh** [obsidianzh/forum#8] [Q&A常见问题]
如题

**Comment by makerjackie** [comment]
目前有[未及时更新的中文帮助文档](https://jackiegeek.gitee.io/obsidian-docs/zh/Obsidian/)，作者估计会在obsidian 1.0 版本再次更新。

有热心的志愿者，也可以到 [官方帮助文档的 github 仓库](https://github.com/obsidianmd/obsidian-docs)提交自己的中文翻译。

**Reply by zen010101** [reply]
https://publish.obsidian.md/help-zh/由此开始

**Comment by xingfu75** [comment]
已经有了
